components/catalyst-ui-ts/divider.tsx

```
@@ -0,0 +1,20 @@
+}: { soft?: boolean } & React.ComponentPropsWithoutRef<'hr'>) {
+  return (
+    <hr
+      role="presentation"
+      {...props}
+      className={clsx(
+        className,
+        'w-full border-t',
+        soft && 'border-zinc-950/5 dark:border-white/5',
+        !soft && 'border-zinc-950/10 dark:border-white/10'
+      )}
+    />
```

Biome (2.1.2)

[error] Interactive elements should not be assigned non-interactive roles.

WAI-ARIA roles should not be used to convert an interactive element to a non-interactive element.
Wrap your interactive element in a <div> with the desired role or put the content inside your interactive element.
Unsafe fix: Remove the role attribute.


(lint/a11y/noInteractiveElementToNoninteractiveRole)